### PR TITLE
run "down" using project if it was set for command

### PR DESCRIPTION
**Pull request by ndeloof**
**What I did**

Pass `Project` to Down if it was set on command line, so we ddon't need to search for compose files

**Related issue**
https://github.com/docker/compose-cli/issues/1141

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**

### CODE REVIEW

File: cli/cmd/compose/compose.go [hunk 1]
```diff
@@ -45,21 +45,21 @@ func addComposeCommonFlags(f *pflag.FlagSet, opts *composeOptions) {
 	f.BoolVarP(&opts.Quiet, "quiet", "q", false, "Only display IDs")
 }
 
-func (o *composeOptions) toProjectName() (string, error) {
+func (o *composeOptions) toProjectName() (string, *types.Project, error) {
```
Comment (gtardif) [inline cli/cmd/compose/compose.go:48]: Should we just return Project here, and name the function toProject() ? the signature is strange

**Review by gtardif — APPROVED** [review]
LGTM with minor improvement on toProjectName that now has a strange signature

File: cli/cmd/compose/compose.go (hunk 1 above)
Comment (ndeloof) [reply cli/cmd/compose/compose.go:48]: I'm not sure about adequate naming here, as this has to return either name or name _and_ project.
Maybe we could use `golang.org/x/functional/Either` :P

Comment (gtardif) [reply cli/cmd/compose/compose.go:48]: My point was : we could return just the project and not the name (that's inside the project). And update the caller to use p.Name (that will not require `if err != nil` in this case :) ).

Comment (ndeloof) [reply cli/cmd/compose/compose.go:48]: sure, but doing so we don't make it clear project has been loaded and is fully set or we just have a name. Backend code would have to check something like `if project.Services == nil` to distingish those scenarios.

**Review by gtardif — APPROVED** [review]

**Pull request merged**